### PR TITLE
Update pallai.txt

### PR DESCRIPTION
**Pull request by Vicky7873**
8th line added for pr please check and suggest

### CODE REVIEW

File: pallai.txt
Comment (bhikiLead) [inline pallai.txt:1]: please tell me why did you add this

**Review by Vicky7873 — COMMENTED** [review]
please check and suggest

File: pallai.txt [hunk 1]
```diff
@@ -3,3 +3,6 @@ try with token
 lets try with git bash
 lets try the merge conflict
 with merge conflict
+
+========================
+New code from dev 1
```
Comment (bhikiLead) [inline pallai.txt:8]: please change the number 1 to 2

**Review by bhikiLead — CHANGES_REQUESTED** [review]

**Review by bhikiLead — CHANGES_REQUESTED** [review]
from 2 to 3

File: pallai.txt [hunk 2]
```diff
@@ -5,4 +5,4 @@ lets try the merge conflict
 with merge conflict
 
 ========================
-New code from dev 1
+New code from dev 2
```
Comment (Vicky7873) [inline pallai.txt:8]: here we need to do the changes

**Review by Vicky7873 — COMMENTED** [review]
please check

File: pallai.txt [hunk 3]
```diff
@@ -3,3 +3,6 @@ try with token
 lets try with git bash
 lets try the merge conflict
 with merge conflict
+
+========================
+New code from dev 2
```
Comment (bhikiLead) [inline pallai.txt:8]: yes their only please change

**Review by bhikiLead — COMMENTED** [review]
yes

File: pallai.txt (hunk 2 above)
Comment (bhikiLead) [reply pallai.txt]: done the chnages

**Review by bhikiLead — APPROVED** [review]
Approved

**Comment by Vicky7873** [comment]
code for line 8
